pep-0683.rst

```diff
@@ -416,9 +412,9 @@ makes it immortal.
 
```

Comment:
In the specification above, the second bullet point, maybe let's remove the part that says: '(or its most significant bit)', I think the sentence before that is specific enough and this parenthesis dives into implementation details